packages/web-components/src/components/filter-panel/filter-group.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement, html, LitElement, internalProperty, property } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import settings from 'carbon-components/es/globals/js/settings';
+import { classMap } from 'lit-html/directives/class-map';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import './input_select';
+
+import styles from './filter-panel.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+@customElement(`${ddsPrefix}-filter-group`)
+class DDSFilterGroup extends StableSelectorMixin(LitElement) {
+  @internalProperty()
+  protected _childItems: Element[] = [];
+
+  @internalProperty()
+  private selectorItem: string = '';
+
+  private _handleSlotChange(event: Event) {
+    this._childItems = (event.target as HTMLSlotElement)
+      .assignedNodes({ flatten: true })
+      .filter(node => node.nodeType === Node.ELEMENT_NODE && (node as Element)?.matches(this.selectorItem)) as Element[];
+  }
+
+  @property()
+  title: string = '';
+
+  @internalProperty()
+  private isOpen: boolean = true;
+
+  protected toggleAccordion() {
+    this.isOpen = !this.isOpen;
+  }
+
+  render() {
+    const { title, isOpen } = this;
+    const classes = classMap({
+      'bx--accordion__item': true,
+      'bx--accordion__item--active': isOpen,
+    });
+    return html`
+      <ul class="${prefix}--accordion">
```

Comment:
should this instead be using carbon-web-components' accordian `<bx-accordion></bx-accordion>`?